src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java

```diff
@@ -374,21 +377,27 @@ private void registerStudentExamMetrics() {
                 .description("Number of exams starting within the next minutes multiplied with students in the course").register(meterRegistry);
     }
 
+    private void registerActiveAdminMetrics() {
+        activeAdminsGauge = MultiGauge.builder("artemis.users.admins.active").description("User logins of active admin accounts").register(meterRegistry);
+    }
+
     /**
      * Calculate active users (active within the last 14 days) and store them in a List.
      * The calculation is performed every 60 minutes.
      * The initial calculation is done in the constructor to ensure it is done BEFORE {@link #recalculateMetrics()}
      * is called.
      */
     @Scheduled(fixedRate = 60 * 60 * 1000, initialDelay = 60 * 60 * 1000) // Every 60 minutes
-    public void calculateCachedActiveUserNames() {
+    public void calculateActiveUserMetrics() {
         var startDate = System.currentTimeMillis();
 
         // The authorization object has to be set because this method is not called by a user but by the scheduler
         SecurityUtils.setAuthorizationObject();
 
         cachedActiveUserNames = statisticsRepository.getActiveUserNames(ZonedDateTime.now().minusDays(14), ZonedDateTime.now());
 
+        updateActiveAdminsMetrics();
+
         log.debug("calculateCachedActiveUserLogins took {}ms", System.currentTimeMillis() - startDate);
```

Comment:
Nit: Also update the log message here to 
```suggestion
        log.debug("calculateActiveUserMetrics took {}ms", System.currentTimeMillis() - startDate);
```